kratos/tests/input_output/test_logger.cpp

```diff
@@ -142,34 +142,34 @@ namespace Kratos {
         KRATOS_TEST_CASE_IN_SUITE(LoggerStreamWarning, KratosCoreFastSuite)
         {
             static std::stringstream buffer;
-			LoggerOutput::Pointer p_output(new LoggerOutput(buffer));
-			Logger::AddOutput(p_output);
+            LoggerOutput::Pointer p_output(new LoggerOutput(buffer));
+            Logger::AddOutput(p_output);
 
-			KRATOS_INFO("TestWarning") << "Test warning message";
+            KRATOS_WARNING("TestWarning") << "Test warning message";
 
             KRATOS_CHECK_C_STRING_EQUAL(buffer.str().c_str(), "TestWarning: Test warning message");
         }
 
         KRATOS_TEST_CASE_IN_SUITE(LoggerStreamWarningIf, KratosCoreFastSuite)
         {
             static std::stringstream buffer;
-			LoggerOutput::Pointer p_output(new LoggerOutput(buffer));
-			Logger::AddOutput(p_output);
+            LoggerOutput::Pointer p_output(new LoggerOutput(buffer));
+            Logger::AddOutput(p_output);
 
-            KRATOS_INFO_IF("TestWarning", true) << "Test warning message";
-            KRATOS_INFO_IF("TestWarning", false) << "This should not appear";
+            KRATOS_WARNING_IF("TestWarning", true) << "Test warning message";
```

Comment:
Shouldn't both variants be tested?
Or is that handled internally through the level?

Reply:
Mm not sure what you mean. Which variant is not tested? I may have missed a test :S

Reply:
You change `KRATOS_INFO_IF` to `KRATOS_WARNING_IF`
Are they both using the same implementation with the only difference that they internally use another severity lvl?

Reply:
Yes

Reply:
For example, if you look at the tests for the "detail" level, I had to specify explicitly the output level, otherwise it wouldn't have printed anything. 

I can make the same for the warning and info levels if it is clearer.